commands/poll.js

```diff
@@ -0,0 +1,151 @@
+const Discord = require('discord.js');
+const emojiRegex = require('emoji-regex');
+
+const symbols = require('../symbols');
+const functions = require('../functions');
+
+function generateTestResults(options) {
+	op = {...options};
+	for (let i = 0; i < Object.keys(op).length; i++) {
+		op[Object.keys(op)[i]] = Math.floor(Math.random() * 100);
+	}
+	return op;
+}
+function generateResults(options) {
+	op = {...options};
+	for (let i = 0; i < Object.keys(op).length; i++) {
+		op[Object.keys(op)[i]] = 0;
+	}
+	return op;
+}
+
+module.exports = {
+	name: 'poll',
+	description: 'Run a poll in a specified channel using reactions',
+	execute(client, message, args) {
+		let poll_options = {};
+		if (args.length > 0) {
+			let guild = client.guilds.cache.get(client.CSIT_ID);
+			let channel = guild.channels.cache.find(ch => ch.name === args.join(' '));
+			if (channel != null) {
+				message.channel.send(`Allow me to assist you with starting a poll in <#${channel.id}> today.\nIf you change your mind and wish to cancel, simply type \`stop\` at any point.`);
+
+				message.channel.send('**Firstly**, what would you like to call the poll? (e.g. "Weekly Trivia Availability")');
+				let stage = 0;
+				options_index = 0;
+				const collector = message.channel.createMessageCollector(m => m.author.id === message.author.id, { time: 300000 });
```

Comment:
That number is big, what is that number, hours, minutes? I can't read that quickly.

Reply:
Milliseconds -> 5 minutes